iconservice/iiss/reward_calc/msg_data.py

```diff
@@ -105,13 +109,17 @@ def from_bytes(key: bytes, value: bytes) -> 'GovernanceVariable':
         data_list: list = MsgPackForIpc.loads(value)
         obj = GovernanceVariable()
         obj.block_height: int = int.from_bytes(key[2:], DATA_BYTE_ORDER)
-        obj.calculated_irep: int = data_list[0]
-        obj.reward_rep: int = data_list[1]
+        obj.config_main_prep_count: int = data_list[0]
```

Comment:
How about backward compatibility?
Can Reward calculator load both of old and new data formats without version info?

Reply:
Icon service does not need to care about backward compatibility as it does not read the data. But reward calculator must consider about backward compatibility.